### DISCUSSION TITLE
Team focus changes & @camunda-community-hub/devrel

### DISCUSSION THREAD

**Discussion by xomiamoore** [camunda-community-hub/community#53] [Announcements]
Hello everyone! Just a few quick updates today. Recently our focus areas for the [DevRel](https://github.com/camunda-community-hub/devrel-team#developer-relations) and [DevEx](https://github.com/camunda-community-hub/devrel-team#developer-relations) teams have shifted, so I wanted to highlight the changes we've made to our [devrel-team page](https://github.com/camunda-community-hub/devrel-team#who-are-we):

>    **Developer Relations** focuses on creating a valuable community experience for Camunda users and encouraging external advocates to spread the word about Camunda. We do this by ensuring people can find solutions to their problems through the forum & content, listening to the feedback from our community and passing it along to the proper teams, and making it easy for community members to contribute code as well as content.

>    **Developer Experience** focuses on external developer enablement for products through documentation and product-adjacent tools at Camunda. Our mission is to lower barriers for all developers working programmatically in and around the Camunda ecosystem to get what they need to be effective with Camunda efficiently.

With this slight change in focus, I wanted to remind everyone that it is best to use **@camunda-community-hub/devrel** when alerting the Camunda team to something that needs a push. This will alert the whole team, so regardless of vacations or any team/employee shuffles, someone will see your request!

**Comment by akeller** [comment]
Thanks for this announcement @xomiamoore!
